typedoc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "telegraf.js"
```

Comment:
Why is this needed? Typedoc seems to infer `"telegraf"`

Reply:
It's already pretty inconsistent with Telegraf vs. telegraf.js. Let's not introduce a third name

Reply:
Fair enough, just move this to `tsconfig`

Reply:
`tsconfig.json` not `package.json`

Reply:
Oops